README.md

```diff
@@ -1,97 +1,100 @@
+```
 
-`aws cloudformation create-stack --stack-name <STACK-NAME> --template-body file://<FILE-PATH>  --region us-east-1 --parameters ParameterKey=BucketNameParameter,ParameterValue=<BUCKET-NAME> --capabilities CAPABILITY_NAMED_IAM`
+### CLI Command Using the URL
 
-#### With URL
+For convenience, the configs are also available on a public S3 bucket, so that you do not need to download them.
 
-`aws cloudformation create-stack --stack-name <STACK-NAME> --template-url 'https://tn-s3-cloud-formation.s3.amazonaws.com/aws-s3-cloud-formation.yaml'  --region us-east-1 --parameters ParameterKey=BucketNameParameter,ParameterValue=<BUCKET-NAME> --capabilities CAPABILITY_NAMED_IAM`
+```term
+aws cloudformation create-stack --stack-name <STACK-NAME> --template-url 'https://tn-s3-cloud-formation.s3.amazonaws.com/aws-s3-cloud-formation.yaml'  --region us-east-1 --parameters ParameterKey=BucketNameParameter,ParameterValue=<BUCKET-NAME> --capabilities CAPABILITY_NAMED_IAM
+```
 
-The following arguments are required: 
+The following arguments are required:
 
-- `--stack-name <STACK-NAME>` this must be unique 
+- `--stack-name <STACK-NAME>` this must be unique
 - `--parameters ParameterKey=BucketNameParameter,ParameterValue=<BUCKET-NAME>` BUCKET-NAME must be unique and lowercased
 - `--capabilities CAPABILITY_NAMED_IAM`
-- `--template-body file://<FILE-PATH>`  path should start with file:// one of `--template-body` or `--template-url`
+- `--template-body file://<FILE-PATH>` path should start with file:// one of `--template-body` or `--template-url`
 - `--template-url <FILE-URL>` one of `--template-body` or `--template-url`
 
 ### Using the AWS Console
-- visit the console, sign in and navigate to the CloudFormation Dashboard
+
+You can also run the "stack" from the AWS Console:
+
+- Visit the console, sign in and navigate to the CloudFormation Dashboard
 - Click create stack (with new resources)
 - select Template is ready
 - Select Amazon S3 URL and provide the yaml file from this repo uploaded to S3 as the [link](https://tn-s3-cloud-formation.s3.amazonaws.com/aws-s3-cloud-formation.yaml)
 - Click next and pass in the required parameter value (S3 Bucket Name)
 
-### Get the appropriate output variables 
+### Get the appropriate output variables
 
-When the cloud formation is done you can get the Access Key Id, Secret and Bucket name from the outputs 
+When the cloud formation is done you can get the Access Key ID, Secret, and Bucket name from the outputs
+
+#### Using the cli
 
-#### Using the cli 
 `aws cloudformation describe-stacks --stack-name <STACK-NAME>` from the previously create command
 
 This will return a json object to retrieve the variables tab down to the `Outputs` key
 
-#### Using the console 
+#### Using the console
 
 Visit the CloudFormation Dashboard, click into the new stack you created and then tap the Outputs Tab
 
-### Video instructions
-https://www.loom.com/share/f335244216264794aa78c0af8afbdffd
```

Comment:
@paribaker - I removed this link, because it appears to be dead. Video instructions would be nice, so if you can dig up a good link to the video, then we can add this back.